media-gfx/alembic/alembic-1.7.9.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Alembic is an open framework for storing and sharing scene data"
+HOMEPAGE="http://alembic.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+
+SLOT="0"
+# TODO: ~x86 currently depends on new =dev-python/pyilmbase-2.3.0 which has
+# ~x86 keyword. As soon as it's updated in the tree, the keyword can be
+# added here.
+KEYWORDS="~amd64"
+IUSE="arnold +boost doc examples hdf5 maya prman python test zlib"
+
+# pyalembic python bindings need boost
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	hdf5? ( zlib )
+	python? ( boost )
+"
+
+CDEPEND="
```

Comment:
Single-letter prefix for `DEPEND` is reserved for future EAPIs. But it is not even required in this case, just make it `RDEPEND` and make `DEPEND` inherit that.

Reply:
Didn't know about the single letter prefixes. Initially I thought I will have more differences, so an extra COMMON_DEPEND would be meaningful, but this isn't actually the case. I changed it.